scripts/js/lib/links/InternalLink.ts

```diff
@@ -114,14 +114,14 @@ export class InternalLink {
     let suggestionPath: String = "";
     let suggestionPathAnchors: string[] = [];
 
-    const possiblePaths = this.possibleFilePaths(originFile);
-    const pathNoExtension = possiblePaths[0].replace(/\.[^\/.]+$/, "");
-
     existingFiles.forEach((file) => {
-      let score = levenshtein.get(pathNoExtension, file.path);
+      const candidatePath = file.path.startsWith("public/")
+        ? file.path.replace(/^public/, "")
+        : file.path.replace(/\.[^\/.]+$/, "").replace(/^docs/, "");
```

Comment:
The first `replace` removes the file extension, which would be wrong to do for `public/`.